lib/system/assertions.nim

```diff
@@ -1,3 +1,7 @@
+## This module provides various assertion utilities.
+##
+## **Note:** This is part of the system module. Do not import it directly.
```

Comment:
```suggestion
## **Note:** This is imported/reexported by `system`.
```
since it's a module, not include, there should be no restriction to import

Reply:
Well, at least importing it in a new project doesn't work for me...

Reply:
works for me (with `import system/assertions`)

Reply:
Ah, that works, I will change the note.